hadoop-ozone/dist/src/main/k8s/examples/ozone/test.sh

```diff
@@ -28,7 +28,17 @@ regenerate_resources
 
 start_k8s_env
 
-execute_robot_test scm-0 smoketest/basic/basic.robot
+execute_robot_test ${SCM} -v PREFIX:pre smoketest/freon/generate.robot
```

Comment:
`SCM` variable is empty.

https://github.com/apache/ozone/runs/6651305707#step:5:737